### PR TITLE
New working software list additions (cdi.xml, ibm5170_cdrom.xml)

### PR DESCRIPTION
**Pull request by einstein95**
The Cranberies: Doors and Windows (USA) [redump.org]

### CODE REVIEW

File: hash/ibm5170_cdrom.xml [hunk 1]
```diff
@@ -1933,6 +1933,34 @@ license:CC0
...
+	<rom name="Cranberries, The - Doors and Windows (USA) (Track 5).bin" size="37269792" crc="26ed1a0d" md5="99afa04401a863b4238425848954a2a1"/>
+	<rom name="Cranberries, The - Doors and Windows (USA) (Track 6).bin" size="55996416" crc="4e78455b" md5="fca6e7e070cfb63d59e5872bd1692034"/>
+	<rom name="Cranberries, The - Doors and Windows (USA).cue" size="821" crc="2df31efa" md5="3875f3b228779ee4a7efce0939cd60c4"/>
+	Note: Track 1 in the cue file had to be edited to read MODE2/2352 to get around chdman not recognizing a valid CDI/2352 value.
+	The info for the original cue is below.
+	<rom name="Cranberries, The - Doors and Windows (USA).cue" size="819" crc="c4424220" md5="cd3b4ef9fe2c410b199e6ceede112cc3"/>
+	-->
+	<!-- TODO: Continuous MAME warnings about "requires ide_controller_32_device" -->
+	<software name="doorswin">
+		<description>The Cranberies: Doors and Windows (USA)</description>
```
Comment (0kmg) [inline hash/ibm5170_cdrom.xml:1953]: Cranberries has two Rs.

File: hash/ibm5170_cdrom.xml [hunk 2]
```diff
@@ -1933,6 +1933,34 @@ license:CC0
...
+		<year>1995</year>
+		<publisher>Phillips Media / Island Records</publisher>
```
Comment (0kmg) [inline hash/ibm5170_cdrom.xml:1955]: Philips has one L.

File: hash/cdi.xml [hunk 3]
```diff
@@ -11106,4 +11106,30 @@ license:CC0
...
+		<rom name="Cranberries, The - Doors and Windows (USA) (Track 5).bin" size="37269792" crc="26ed1a0d" md5="99afa04401a863b4238425848954a2a1"/>
+		<rom name="Cranberries, The - Doors and Windows (USA) (Track 6).bin" size="55996416" crc="4e78455b" md5="fca6e7e070cfb63d59e5872bd1692034"/>
+		<rom name="Cranberries, The - Doors and Windows (USA).cue" size="821" crc="2df31efa" md5="3875f3b228779ee4a7efce0939cd60c4"/>
+		Note: Track 1 in the cue file had to be edited to read MODE2/2352 to get around chdman not recognizing a valid CDI/2352 value.
+		The info for the original cue is below.
+		<rom name="Cranberries, The - Doors and Windows (USA).cue" size="819" crc="c4424220" md5="cd3b4ef9fe2c410b199e6ceede112cc3"/>
+		-->
+		<description>The Cranberies: Doors and Windows (USA)</description>
+		<year>1995</year>
+		<publisher>Phillips Media / Island Records</publisher>
```
Comment (0kmg) [inline hash/cdi.xml:11126]: These guys too.

File: hash/cdi.xml [hunk 4]
```diff
@@ -11106,4 +11106,30 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="doorswin">
+		<!-- Rainbow disc (CD-ROM XA / CD-i / VideoCD / CD-DA) -->
```
Comment (FoxhackDN) [inline hash/cdi.xml:11110]: Does this run fine without the DVC cartridge? If so it might be worth adding a note about it.

Comment (einstein95) [reply hash/cdi.xml]: Note added

File: hash/cdi.xml (hunk 3 above)
Comment (einstein95) [reply hash/cdi.xml]: Fixed

File: hash/ibm5170_cdrom.xml (hunk 2 above)
Comment (einstein95) [reply hash/ibm5170_cdrom.xml]: Fixed

File: hash/ibm5170_cdrom.xml (hunk 1 above)
Comment (einstein95) [reply hash/ibm5170_cdrom.xml]: Fixed

**Comment by cuavas** [comment]
Do we have a picture of the label side of the CD?  This was released in a “limited edition box set” as catalog number 810 5107.  Do we know what else was in the set, and what the labels of any other included discs said?

**Comment by FoxhackDN** [comment]
> Do we have a picture of the label side of the CD? This was released in a “limited edition box set” as catalog number 810 5107. Do we know what else was in the set, and what the labels of any other included discs said?

There are multiple photos of the release [on Discogs](https://www.discogs.com/release/1437408-The-Cranberries-Doors-And-Windows), but none of the boxset. However, [this page](https://es.wallapop.com/item/cofre-cranberries-ep-doors-and-w-cd-cd-rom-768570745) shows a photo of an European version of the set and it's just the CD in a box with a little coin purse.

That version was published by Island Records and Philips according to Discogs, but we have no way of being sure if the data is exactly the same until someone dumps it... it might be the same data for all we know.

**Comment by 0kmg** [comment]
@FoxhackDN FYI, that last link is NOT photos only. Here's an ebay listing of the same disc with matching catalog numbers instead: https://www.ebay.com/itm/264842626867.

**Comment by ArcadeShadow** [comment]
Try this [link](https://www.theworldofcdi.com/cd-i_encyclopedia/the-cranberries-doors-and-windows/), maybe it can help.

**Comment by einstein95** [comment]
@cuavas image

**Comment by cuavas** [comment]
I guess this can be merged.  At some point we’re going to need to need a “multimedia CDs” software list or something for these CDs that were intended to be played on several generally incompatible systems.

**Pull request merged**